nixos/modules/system/activation/switch-to-configuration.pl

```diff
@@ -350,76 +390,78 @@ sub compare_units {
     return $ret;
 }
 
-sub handleModifiedUnit {
-    my ($unit, $baseName, $newUnitFile, $newUnitInfo, $activePrev, $unitsToStop, $unitsToStart, $unitsToReload, $unitsToRestart, $unitsToSkip) = @_;
+# Called when a unit exists in both the old systemd and the new system and the units
+# differ. This takes care of what units are to be stopped, restarted, reloaded, started, and skipped.
```

Comment:
I don't really understand the meaning of this. Does it "take care" in terms of taking action? If so:

```suggestion
# differ. This takes of stopping, restarting, reloading, starting, and skipping units.
```